sample/TKNotificationPollSamples.m

```diff
@@ -0,0 +1,77 @@
+//
+//  TKNotificationPollSamples.m
+//  TokenSdkTests
+//
+//  Created by Larry Hosken on 11/8/17.
+//  Copyright © 2017 Token Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "TKTestBase.h"
+
+#import "TokenSdk.h"
+
+@interface TKNotificationPollSamples : TKTestBase
+
+@end
+
+@implementation TKNotificationPollSamples
+
+-(void)testNotificationPolling {
+    TokenIOSync *tokenIOSync = [[self sdkBuilder] buildSync];
+    Alias *payerAlias = [self generateAlias];
+    TKMemberSync *payerSync = [tokenIOSync createMember:payerAlias];
+    [payerSync linkAccounts:[self createBankAuthorization:payerSync]];
+    TKMember *payer = payerSync.async;
+    
+    __block Subscriber *subscriber = nil;
+    
+    [payer subscribeToNotifications:@"iron"
+                handlerInstructions:(NSMutableDictionary<NSString *, NSString *> *) [NSMutableDictionary dictionary]
```

Comment:
For samples, I think we don't want to share the subscription methods since this is only for our ios/Android app. For external developer's app, they cant leverage our apns or fcm service since we don't have their push certs.

Reply:
devs don't have a reason to subscribe "real" members (who use the app), but do for test members. If you try to notify a member with no subscribers, the notification fails. For example, this sample code would not work: notification never arrives because recipient has no subscribers.

(banks who want to use integration sdk and also want to use Token-y functions in their own apps also have a reason to subscribe. But that's in the bank docs, and we show a handlerInstructions with actual instructions)

Reply:
understood